doc/ref_arch/kubernetes/chapters/chapter01.md

```diff
@@ -47,25 +47,24 @@ According to CNCF TOC (Technical Oversight Committee), following is the definiti
 
 >“Cloud native technologies empower organizations to build and run **scalable** applications in modern, **dynamic environments** such as public, private, and hybrid clouds. Containers, **service meshes**, **microservices**, **immutable infrastructure**, and **declarative APIs** exemplify this approach.
 
->These techniques enable **loosely coupled** systems that are **resilient**, **manageable**, and **observable**. Combined with **robust automation**, they allow engineers to make high-impact changes frequently and predictably with minimal toil.
+>These techniques enable **loosely coupled** systems that are **resilient**, **manageable**, and **observable**. Combined with **robust automation**, they allow engineers to make **high-impact changes frequently and predictably** with minimal toil.
 
 >The Cloud Native Computing Foundation seeks to drive adoption of this paradigm by fostering and sustaining an ecosystem of open source, vendor-neutral projects. We democratize state-of-the-art patterns to make these innovations accessible for everyone”
 
-The definition above is very application centric and look at cloud native from the application point of view, in here, we will try and relate those definitions to the infrastructure on which cloud native applications run. For the purposes of this document, a Cloud Native Network Function (CNF) is a type of cloud native application.
-
-- **scalable**:
-- **dynamic environments**:
-- **service meshes**:
-- **microservices**:
-- **immutable infrastructure**:
-- **declarative APIs**:
-- **loosely coupled**:
-- **resilient**:
-- **manageable**:
-- **observable**:
-- **robust automation**:
-
-> The CNCF TUG is also working on a set of Cloud Native Principles at more detail than the existing CNCF definition: [Expanded Cloud Native Principles](https://github.com/vulk/cloud-native-principles/blob/master/cloud-native-principles.md).  Many of the principles above are taken from this document.
+The CNCF TUG (Telecom User Group) is also working on a set of Cloud Native Principles at more detail than the existing CNCF definition: [Expanded Cloud Native Principles](https://github.com/vulk/cloud-native-principles/blob/master/cloud-native-principles.md). There are many similarities, but the key principles from both, which are applicable to this document, are:
```

Comment:
Can you use a relative url?

Does cloud native require deployment on a cloud infrastructure, in particular, the cloud characteristic of being "shared"? While cloud native workloads need to have all of these listed properties, workloads with these properties doesn't make them cloud-native (most of these principles have been around since before "cloud environments").

Reply:
Not sure if relative link will work as it's a different org/repo but will try.

Reply:
> Does cloud native require deployment on a cloud infrastructure, in particular, the cloud characteristic of being "shared"? While cloud native workloads need to have all of these listed properties, workloads with these properties doesn't make them cloud-native (most of these principles have been around since before "cloud environments").

Not sure about this- I'd argue that whilst the concepts may have been around, they weren't practised - i.e. why we have huge monolithic workloads that take a year to upgrade.  I couldn't name a workload that shows these behaviours today without it also being cloud native.

Reply:
This is not in the [CNCF TUG-s repo](https://github.com/cncf/telecom-user-group), but in Vulk Coops, so I would not consider it to the TUG. Maybe @taylor can explain more how is this relevant.

Reply:
Oh I hadn't noticed that. @taylor can we move it to the TUG repo?

Reply:
@tomkivlin the wording should be updated and the [repo](https://github.com/cloud-native-principles/cloud-native-principles/blob/master/cloud-native-principles.md#cloud-native-principles) has changed.  I recommend the [gitbook url](https://networking.cloud-native-principles.org/) vs the repo.  An update on wording could look something like:

> Also, in collaboration with Cloud native developers, some Telecom operators and vendors have been working on a set of Cloud Native Principles at more detail than the existing CNCF definition: [Expanded Cloud Native Principles](https://github.com/cloud-native-principles/cloud-native-principles/blob/master/cloud-native-principles.md#cloud-native-principles).  This was a precursor to the new TUG prologue and provides references too many of the cloud native concepts and processes.  There are many similarities, but the key principles from both, which are applicable to the this document (RA2 Ch1), are:

FYI, the 5 cloud native principles documents were started independently of and prior to TUG's whitepaper efforts, by some Telecom operators.  The original documents were in google doc format and presented to the TUG in 2019.  They were converted to markdown and it was temporarily hosted by Vulk on github.

Reply:
@tomkivlin @taylor - Is there anything that is missing in the TUG document that we should add it here?

Reply:
I've updated the link @CsatariGergely can you review again please?

Reply:
@pgoyal01 - are you ok for the conversation to be resolved please?

Reply:
> @tomkivlin @taylor - Is there anything that is missing in the TUG document that we should add it here?

I would suggest adding it in the TUG one first, then having a separate PR to update this section.

Reply:
@tomkivlin I am OK. @CsatariGergely can resolve if he is ok with it.